### PR TITLE
HTS-1002 Added Not Eligible Scenarios

### PR DESCRIPTION
**Pull request by sophie-still**

### CODE REVIEW

File: selenium-system-test/src/test/resources/features/checkEligibility/Applicant goes through eligibility check.feature [hunk 1]
```diff
@@ -1,27 +1,42 @@
...
     When they apply for Help to Save
-    Then they see a technical error page
-
-  @HTS-906
-  Scenario: User is NOT eligible and wishes to view appeals page
-    Given a user is NOT in receipt of working tax credit
-    When they apply for Help to Save when NOT eligible
-    Then they see that they are NOT eligible for Help to Save
-    When they then click on still think you're eligible link
-    Then they see appeals and tax tribunal page
```
Comment (sureshhmrc) [inline selenium-system-test/src/test/resources/features/checkEligibility/Applicant goes through eligibility check.feature]: why did we remove the above testcase ? no longer required ?

File: selenium-system-test/src/test/resources/features/checkEligibility/Applicant goes through eligibility check.feature [hunk 2]
```diff
@@ -1,27 +1,42 @@
...
 Feature: Applicant goes through eligibility check
 
-  Scenario: User is in receipt of working tax credit and so is eligible
-    Given a user is in receipt of working tax credit
+  Scenario: User is in receipt of WTC and so is eligible
+    Given a user is in receipt of WTC
     When they apply for Help to Save
     Then they see that they are eligible for Help to Save
 
   @BrowserStack
```
Comment (sureshhmrc) [inline selenium-system-test/src/test/resources/features/checkEligibility/Applicant goes through eligibility check.feature]: just curious, why only NON eligible scenerios have `@BrowserStack` annotation ?

File: selenium-system-test/src/test/resources/features/checkEligibility/Applicant goes through eligibility check.feature (hunk 1 above)
Comment (sophie-still) [reply selenium-system-test/src/test/resources/features/checkEligibility/Applicant goes through eligibility check.feature]: This was removed and replaced by all the possible Not Eligible scenarios. 
The test used a generic "not eligible" NINO and only covered one reason code.

File: selenium-system-test/src/test/resources/features/checkEligibility/Applicant goes through eligibility check.feature (hunk 2 above)
Comment (sureshhmrc) [reply selenium-system-test/src/test/resources/features/checkEligibility/Applicant goes through eligibility check.feature]: i guess, the annotation is applicable to all scenerios in the file ? check with Marcus if its required at all and if needed it  should be added at the top

**Pull request merged**